bazel/external/googleurl.patch

```diff
@@ -76,6 +76,32 @@ index 4a9494e..ba44fa0 100644
      // Force a compile failure if instantiated.
      return CheckOnFailure::template HandleFailure<Dst>();
    }
+diff --git a/base/containers/span.h b/base/containers/span.h
```

Comment:
Can you back-port these changes upstream?

Reply:
Line 45 in this file mentions:
> \# TODO(keith): Remove unused parameter workarounds when https://quiche-review.googlesource.com/c/googleurl/+/11180 lands

@vasilvv what's our plan for unused parameter warnings in QUICHE's version of googleurl?